Sources/SpeziAccessGuard/AccessGuardButton.swift

```diff
@@ -67,11 +67,21 @@ public struct AccessGuardButton<Locked: View, Unlocked: View>: View {
                 locked()
             }
             .sheet(isPresented: $isShowingUnlockSheet) {
-                AccessGuarded(identifier) {
-                    VStack { }
-                        .onAppear {
-                            isShowingUnlockSheet = false
+                NavigationStack {
+                    AccessGuarded(identifier) {
+                        VStack { }
+                            .onAppear {
+                                isShowingUnlockSheet = false
+                            }
+                    }
+                    .navigationTitle(NSLocalizedString("UNLOCK_TITLE", bundle: .module, comment: ""))
```

Comment:
Same intention comments as for the other elements of the PR; always good to indent modifiers by 4 spaces.